### PR TITLE
Add packaging logic

### LINKED ISSUE #5: Add packaging logic
The repo is missing a `setup.py`, `setup.cfg` etc.

### PR DESCRIPTION
**Pull request by martinRenou**
Fix #5

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    Programming Language :: Python :: 3.10
+    Framework :: Jupyter
+
+[options]
+zip_safe = False
+include_package_data = True
+packages = find:
+python_requires = >=3.6
+install_requires =
+    jupyterlab>=3.2,<5
```
Comment (martinRenou) [inline setup.cfg]: wrong version, <4

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	jupyterlab_server~=2.3
+	jupyter_server~=1.4
+	nbclassic~=0.2
+	tornado>=6.1.0
```
Comment (martinRenou) [inline setup.cfg]: tabs instead of spaces

**Comment by jtpio** [comment]
Nice thanks!

Maybe we could also make it compatible with the Jupyter Releaser from the start. That would mean adding other files like CHANGELOG.md and the check-release CI workflow.

So folks starting from this repo can bootstrap their app and release a first version very quickly.

**Comment by jtpio** [comment]
Looks like it picked up a small conflict.

**Comment by martinRenou** [comment]
@jtpio all done :)

**Comment by martinRenou** [comment]
I don't understand the failure of the check-release job, is it complaining because there are no tags on the repo for now?

**Comment by jtpio** [comment]
Ah strange I thought this was now fixed by https://github.com/jupyter-server/jupyter_releaser/pull/261.

File: .github/workflows/check-release.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        run: pip install --upgrade jupyter-packaging~=0.10 --user
+
+      - name: Install Dependencies
+        run: pip install .
+
+      - name: Check Release
+        uses: jupyter-server/jupyter_releaser/.github/actions/check-release@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          version_spec: next
```
Comment (jtpio) [inline .github/workflows/check-release.yml]: The CI could be related to this spec, so we can try without it:

```suggestion
```

**Comment by martinRenou** [comment]
Thanks!

File: pyproject.toml [hunk 4]
```diff
@@ -0,0 +1,20 @@
+[build-system]
+requires = ["jupyter_packaging~=0.10", "jupyterlab>=3.2,<5"]
```
Comment (jtpio) [inline pyproject.toml]:
```suggestion
requires = ["jupyter_packaging~=0.10", "jupyterlab>=3.2,<4"]
```

**Review by jtpio — APPROVED** [review]
Thanks!

**Pull request merged**